automation/Extension/firefox/content.js/index.js

```diff
@@ -1,4 +1,4 @@
 import { injectJavascriptInstrumentPageScript } from "openwpm-webext-instrumentation";
 
-injectJavascriptInstrumentPageScript(true);
-
+const testing = window.openWpmTesting || false;
```

Comment:
I don't think you need to attach testing to `window`. From what I can tell, the two `registerContentScript` calls should inject into the same execution context. You should just be able to define this variable directly in the script registration.

Reply:
Yes, that works. Much better :)